engine.js

```diff
@@ -203,6 +177,13 @@ HMDAEngine.prototype.getFileSpec = function(year) {
     return hmdaRuleSpec.getFileSpec(year);
 };
 
+/**
+ * Get the progress object used for task completion events displayed on the progress bar
+ * @return {object} Progress object containing an eventemitter for progress notification
+ */
+HMDAEngine.prototype.getFileProgress = function() {
+    return hmdajson.getProgress();
+};
```

Comment:
Because this is a convenience function that exposes `getProgress` from `lib/hmdaJson`, since the UI has no direct access to `lib/hmdaJson`. Much like we do for exposing functions from the rule spec module.